sunpy/map/sources/sdo.py

```diff
@@ -121,4 +123,28 @@ def observatory(self):
     @classmethod
     def is_datasource_for(cls, data, header, **kwargs):
         """Determines if header corresponds to an HMI image"""
-        return str(header.get('instrume', '')).startswith('HMI')
+        return (str(header.get('TELESCOP', '')).endswith('HMI') and
+                str(header.get('CONTENT', '')) !=
+                'Carrington Synoptic Chart Of Br Field')
+
+class HMISynopticMap(HMIMap):
+    
+    def __init__(self, data, header, **kwargs):
+        GenericMap.__init__(self, data, header, **kwargs)
```

Comment:
This should be:
```suggestion
        super().__init__(data, header, **kwargs)
```
(also this needs changing in HMIMap)